chirpstack/src/integration/mod.rs

```diff
@@ -28,6 +28,7 @@ pub mod mock;
 mod mqtt;
 mod mydevices;
 mod pilot_things;
+#[cfg(feature = "postgres")]
```

Comment:
Indeed. It may be interesting to reduce dependencies if postgres integration is not desired.

What do you think about gating some integrations under a flag like `integration-postgres`?

Reply:
I think for now lets leave it as-is, as it would also open the discussion if we should put the other integrations between feature flags. Lets first focus on implementing SQLite :)